app/src/main/java/com/yterletskyi/happyfriend/MainActivity.kt

```diff
@@ -35,6 +43,13 @@ class MainActivity : AppCompatActivity() {
             }
         navController.addOnDestinationChangedListener(onDestinationChangeListener)
         view.navBar.setupWithNavController(navController)
+
+        if (pinCodeController_.pinCode.pin.equals(null)) {
```

Comment:
in kotlin we can use just `==` to check equality. To check if it's the same object there is `===` operator.